new-lamassu-admin/src/components/layout/Header.js

```diff
@@ -56,15 +57,27 @@ const Subheader = ({ item, classes }) => {
   )
 }
 
+const notNil = R.compose(R.not, R.isNil)
+
 const Header = memo(({ tree }) => {
   const [open, setOpen] = useState(false)
-  const [anchorEl, setAnchorEl] = React.useState(null)
+  const [anchorEl, setAnchorEl] = useState(null)
+  const [notifButtonCoords, setNotifButtonCoords] = useState({ x: 0, y: 0 })
   const [active, setActive] = useState()
-  const { data, refetch } = useQuery(HAS_UNREAD)
-  const hasUnread = data?.hasUnreadNotifications ?? false
+  const [hasUnread, setHasUnread] = useState(false)
+
+  const { data, refetch } = useQuery(HAS_UNREAD, { pollInterval: 60000 })
+  const notifCenterButtonRef = useRef()
+  const popperRef = useRef()
   const history = useHistory()
   const classes = useStyles()
 
+  useEffect(() => {
+    if (data?.hasUnreadNotifications) return setHasUnread(true)
+    // if not true, make sure it's false and not undefined
+    if (notNil(data?.hasUnreadNotifications)) return setHasUnread(false)
+  }, [data])
```

Comment:
I presume this is for the bell icon on the header being lit up or not. Not the biggest fan of this approach, I think you can reach the same result without using the useEffect hook, but I'll pass it and if Rafael finds it alright, so it is

Reply:
Yeah, since hasUnread becase a state variable I don't think there's a way out of this